code/datums/status_effects/buffs/stop_drop_roll.dm

```diff
@@ -0,0 +1,66 @@
+/datum/status_effect/stop_drop_roll
+	id = "stop_drop_roll"
+	alert_type = null
+
+	tick_interval = 0.8 SECONDS
+
+/datum/status_effect/stop_drop_roll/on_apply()
+	if(!iscarbon(owner))
+		return FALSE
+
+	var/actual_interval = initial(tick_interval)
+	if(!owner.Knockdown(actual_interval * 2, ignore_canstun = TRUE) || owner.body_position != LYING_DOWN)
```

Comment:
is `tick_interval` modified by the status_effect code? that's wack. not really a code concern as much as it is trying to learn what's goin on here

Reply:
yeah, tick interval becomes `world.time + tick_interval` on creation